frontend/src/APIClients/ResidentAPIClient.ts

```diff
@@ -73,6 +74,13 @@ const createResident = async ({
     );
     return true;
   } catch (error) {
+    const axiosErr = (error as any) as AxiosError;
+
+    if (axiosErr.response && axiosErr.response.status === 409) {
+      return {
+        errMessage: "Resident with the specified user ID already exists."
```

Comment:
Here, we're propagating the error message displayed in the toast through the API client